src/sb/app/project/ui.cljc

```diff
@@ -102,12 +104,10 @@
   [:div.field-wrapper
    [:div.field-label (t :tr/team)]
    [:div.grid.grid-cols-2.gap-6
-    (for [member (->> (:membership/_entity project)
-                      (sort-by u/compare:desc :entity/created-at))
-          :let [board-membership (-> member :membership/member)
-                {:as account :keys [account/display-name]} (-> board-membership :membership/member)]]
+    (for [board-membership (member.data/members project (xf/sort-by :entity/created-at u/compare:desc))
```

Comment:
will this will fail because `:membership/member` is already called inside `member.data/members`?